src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js

```diff
@@ -330,7 +330,10 @@ export const CloudEnvironmentModal = ({
           onClick: () => {
             ((toolLabel === tools.Jupyter.label || toolLabel === tools.RStudio.label) && currentRuntime?.status === 'Stopped' ?
               () => startApp(toolLabel) :
-              () => {})()
+              () => {
+                Ajax().Metrics.captureEvent(Events.analysisLaunch,
+                  { origin: 'contextBar', source: toolLabel, application: toolLabel, workspaceName, namespace })
+              })()
             onDismiss()
           }
```

Comment:
Could this be simplified as

```javascript
          onClick: () => {
            toolLabel === tools.Jupyter.label || toolLabel === tools.RStudio.label && currentRuntime?.status === 'Stopped' ?
              startApp(toolLabel) :
              Ajax().Metrics.captureEvent(Events.analysisLaunch,
                { origin: 'contextBar', source: toolLabel, application: toolLabel, workspaceName, namespace })
            onDismiss()
          }
```

?

Reply:
Also the [same comment as above](https://github.com/DataBiosphere/terra-ui/pull/2834/files#r814352929)